engine/apps/grafana_plugin/helpers/client.py

```diff
@@ -162,6 +162,7 @@ def request_headers(self) -> APIRequestHeaders:
 class GrafanaAPIClient(APIClient):
     GRAFANA_INCIDENT_PLUGIN = "grafana-incident-app"
     GRAFANA_INCIDENT_PLUGIN_BACKEND_URL_KEY = "backendUrl"
+    GRAFANA_LABEL_PLUGIN = "grafana-labels-app"
```

Comment:
```suggestion
    GRAFANA_LABELS_PLUGIN = "grafana-labels-app"
```

I propose to name it similar to the app

Reply:
Fixed